src/core/jsonschema/walker.cc

```diff
@@ -1,3 +1,4 @@
+#include <optional>
```

Comment:
```suggestion
```

I don't think you need this include, plus its also in the incorrect place, and without the usual comment